pkg/manager/manager.go

```diff
@@ -64,10 +64,10 @@ func Start(cfg *Config) error {
 		return err
 	}
 
-	enableWebHooks := os.Getenv("ENABLE_WEBHOOKS") != "false"
+	enableWebHooks := os.Getenv("ENABLE_WEBHOOKS") != "false" // TODO: add to config
```

Comment:
Is this a TODO for this PR?

Reply:
For separate refactoring. There are no functional implications, obviously.